lib/Parse/ParseDecl.cpp

```diff
@@ -851,8 +850,8 @@ Parser::parseDifferentiableAttribute(SourceLoc atLoc, SourceLoc loc) {
   }
 
   return ParserResult<DifferentiableAttr>(
-    DifferentiableAttr::create(Context, /*implicit*/ false, atLoc,
-                               SourceRange(loc, rParenLoc),
+      DifferentiableAttr::create(Context, /*implicit*/ false, atLoc,
+                               SourceRange(loc, rParenLoc), linear,
```

Comment:
Indent with line above, same for line below:
```suggestion
                                 SourceRange(loc, rParenLoc), linear,
                                 params, jvpSpec, vjpSpec, whereClause));
```

Reply:
I caught the fact it was using 2 spaces instead of 4 but ya forgot about intending the parameters